From: Robert Ancell <robert.ancell@canonical.com>
Date: Mon, 10 Sep 2018 01:58:43 +0000
Subject: region: Fix language section not being marked insensitive

This is due to the widget not being extracted from region.ui and was introduced
in 5c418b898318dbd8504fcdbe800381244bb54e9c.

(cherry picked from commit d4340c4efcf9d75c2036dea2c8a59f0938bc62f3)

Origin: upstream, 3.30.1
---
 panels/region/cc-region-panel.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/panels/region/cc-region-panel.c b/panels/region/cc-region-panel.c
index 2b71223..0023e3a 100644
--- a/panels/region/cc-region-panel.c
+++ b/panels/region/cc-region-panel.c
@@ -1654,6 +1654,7 @@ cc_region_panel_class_init (CcRegionPanelClass * klass)
 
         gtk_widget_class_bind_template_child (widget_class, CcRegionPanel, language_row);
         gtk_widget_class_bind_template_child (widget_class, CcRegionPanel, language_label);
+        gtk_widget_class_bind_template_child (widget_class, CcRegionPanel, language_section);
         gtk_widget_class_bind_template_child (widget_class, CcRegionPanel, formats_row);
         gtk_widget_class_bind_template_child (widget_class, CcRegionPanel, formats_label);
         gtk_widget_class_bind_template_child (widget_class, CcRegionPanel, restart_revealer);
